<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ma_md</class>
 <widget class="QMainWindow" name="ma_md">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criterio binario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="6">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Rasgos</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Criterio de Comparaci√≥n</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor umbral </string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="OK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>mayor que un umbral</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>igualdad estricta</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Definir a todos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
